ticdc/manage-ticdc.md

```diff
@@ -69,6 +69,32 @@ tiup cluster upgrade <cluster-name> v4.0.15
 
 ### 管理同步任务 (`changefeed`)
 
+#### 同步任务状态流转
+本功能适用于 4.0.16 及更新版本。
```

Comment:
```suggestion

本功能适用于 4.0.16 及更新版本。
```

Reply:
This might be unrelated to versions. @leoppro PTAL, thanks